content/en/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -19,9 +19,16 @@ Using the latest compatible version of kubectl helps avoid unforeseen issues.
 
 The following methods exist for installing kubectl on Linux:
 
-- [Install kubectl binary with curl on Linux](#install-kubectl-binary-with-curl-on-linux)
-- [Install using native package management](#install-using-native-package-management)
-- [Install using other package management](#install-using-other-package-management)
+- [{{% heading "prerequisites" %}}](#-heading-prerequisites-)
```

Comment:
Remove this line (I think it sneaker in by mistake)

Reply:
Fixed